data-esnext.js

```diff
@@ -778,6 +778,7 @@ exports.tests = [
     babel: true,
     jsx: true,
     chrome58: 'flagged',
+    typescript: typescript.corejs,
```

Comment:
It seems no:
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#object-spread-and-rest